libfuse/constants.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libfuse
+
+import "time"
+
+// QuotaUsageStaleTolerance is the lifespan of stale usage data that libfuse
+// accetps in the Statfs handler. In other words, this causes libkbfs to issue
+// a fresh RPC call if cached usage data is older than 10s.
+const QuotaUsageStaleTolerance = 10 * time.Second
```

Comment:
I think you could put this in `fs.go` rather than have a whole file for it (it also doesn't need to be exported).